HerPortal.Data/DataAccessProvider.cs

```diff
@@ -17,6 +17,7 @@ public async Task<User> GetUserByEmailAsync(string emailAddress)
     {
         var users = await context.Users
             .Include(u => u.LocalAuthorities)
+            .Include(u => u.Consortia)
```

Comment:
It's probably not worth worrying a ton about here, but it would be a good idea to check the places that are using this method to see if they're actually using `Consortia`. If not then we're joining unnecessarily.

Reply:
Probably a good one for us to chat about if you're not sure what's going on with this!

Reply:
I've just checked, and in this instance, we do access consortia when compiling our CSVs :)